addons/point_of_sale/models/pos_order.py

```diff
@@ -479,6 +479,16 @@ def _get_rounded_amount(self, amount):
         currency = self.currency_id
         return currency.round(amount) if currency else amount
 
+    def get_partner_bank_id(self):
```

Comment:
@andro19951 Sorry I forgot to tell you. Could you rename the function "_get_partner_bank_id". As we don't need to call it from javascript and/or as an action on a button in a view, it is better to make it private, it can avoid possible security leaks (no leak in this case, but it is a good practice).

Reply:
done!